editor/src/clj/editor/app_view.clj

```diff
@@ -1956,22 +1961,25 @@
         "Defold"))
   ([project-title] (str project-title " - " (make-title))))
 
-(defn- refresh-app-title! [^Stage stage project]
-  (let [settings      (g/node-value project :settings)
-        project-title (settings ["project" "title"])
-        new-title     (make-title project-title)]
+(defn- refresh-app-title! [^Stage stage project evaluation-context]
+  (let [project-title (some-> (g/maybe-node-value project :settings evaluation-context)
+                              (get ["project" "title"]))
+        new-title (if project-title
+                    (make-title project-title)
+                    (make-title))]
```

Comment:
Note: We now use `maybe-node-value` to evaluate the project title. This avoids a `NullPointerException` if `game.project` is corrupted.